devfile.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: 1.0.0
+
+metadata:
+  generateName: che-plugin-registry-dev-
+
+
+components:
+
+  - mountSources: true
```

Comment:
I would say we don't need this container by default as you have git in the IDE (or add it inside the plugin-dev image)

Reply:
well, I use it and it is not harmful IMHO, it contains not just git but also:
- ability to use `git add -p`
- `tig`
- shell prompt with branch
and it is reusable else where.

Reply:
@benoitf I have removed the container from the devfile.yaml